TR-4558: Update CloudFlare HTTPS setup instructions (#740)
SparkPost/support-docs: content/docs/tech-resources/enabling-https-engagement-tracking-on-sparkpost.md

---
## Step by Step Guide with CloudFlare

> _Updated June 2023. Uses a simpler forwarding method without the need for custom page rules. Images and descriptions follow the current CloudFlare web UI._

Reviewer: Small suggestion so the text is the same as the CloudFront one. I don't think the additional information is useful.
```suggestion
_Updated for Cloudflare web UI as of June 2023._
```

Author: Done
